Etapa1/src/com/company/library/BigLibrary.java

```diff
@@ -0,0 +1,189 @@
+package com.company.library;
+
+import com.company.offer.BigOffer;
+import com.company.offer.KidsOffer;
+import com.company.offer.Offer;
+import com.company.product.Book;
+import com.company.product.ChildBook;
+import com.company.product.Manual;
+import com.company.product.Novel;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Scanner;
+
+public class BigLibrary extends Library{
+    private List<Novel> novels;
+    private List<BigOffer> offers;
+    private List<Manual> manuals;
+
+    public BigLibrary() {
+        this.novels = new ArrayList<Novel>();
+        this.offers = new ArrayList<BigOffer>();
+        this.manuals = new ArrayList<Manual>();
+    }
+
+    public BigLibrary(String name, double rating, HashMap<String, Integer> stock, List<Novel> novels, List<BigOffer> offers, List<Manual> manuals) {
+        super(name, rating, stock);
+        this.novels = novels;
+        this.offers = offers;
+        this.manuals = manuals;
+    }
+
+    @Override
+    public void read(){
```

Comment:
ai amestecat metode de service in clasa de entitate. BigLibrary ar trebui sa fie POJO, doar cu campuri, constructor si getteri-setteri (optional equals, toString)